SDKAndroid/src/main/java/app/cybrid/sdkandroid/components/accounts/view/AccountsViewModel.kt

```diff
@@ -142,17 +146,41 @@ class AccountsViewModel : ViewModel() {
         }
     }
 
-    fun getTradeAmount(trade: TradeBankModel, assets:List<AssetBankModel>) : String {
+    fun getTradeAmount(trade: TradeBankModel, assets:List<AssetBankModel>?) : String {
 
         val tradeSymbol = trade.symbol
         val assetsParts = tradeSymbol?.split("-")
         val assetString = assetsParts!![0]
-        val asset = assets.find { it.code == assetString }
+        val asset = assets?.find { it.code == assetString }
         val returnValue = if (trade.side == TradeBankModel.Side.sell) {
-            BigDecimalPipe.transform(BigDecimal(trade.deliverAmount!!), asset!!)
+            AssetPipe.transform(BigDecimal(trade.deliverAmount!!), asset!!, "trade")
         } else {
+            AssetPipe.transform(BigDecimal(trade.receiveAmount!!), asset!!, "trade")
```

Comment:
Instead of using the string `trade` we should use a defined constant.

Reply:
Done!